app/src/components/Layout.tsx

```diff
@@ -0,0 +1,24 @@
+import { useTranslation } from 'next-i18next'
+import { ReactElement } from 'react'
+
+type Props = {
+  children: ReactElement
+}
+
+const Layout = ({ children }: Props): ReactElement => {
+  const { t } = useTranslation('common')
+
+  return (
+    <div className="container">
```

Comment:
nit: eventually, it'd be nice to switch over to using USWDS classes rather than encourage custom classes for things that are in the design system already (typography, [header](https://designsystem.digital.gov/components/header/), [footer](https://designsystem.digital.gov/components/footer/), spacing). For instance, I think we could use [`grid-container` and other grid classes](https://designsystem.digital.gov/utilities/layout-grid/) for the main wrapper.

Reply:
Good call. Let's pull this out of this PR and I'll make a separate ticket for this.